cmd/root.go

```diff
@@ -285,6 +278,38 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 	return nil
 }
 
+// parseBoolOpt parses a boolean option from the query string
+//  name the name of the option in the query string
+//  q the map of the parsed query string parameters from url.ParseQuery()
+// returns
+//   nil if no value was found
+//   true if the value is 't' or 'true' case insensitive
+//   false if the value is 'f' or 'false' case insensitive
+func parseBoolOpt(name string, q map[string][]string) (*bool, error) {
+	if iam, ok := q[name]; ok {
```

Comment:
This should be `!ok`. Typically the error flow is indented and the happy path is not.